### ISSUE TITLE
unable to open in unreal 4.26.2

### ISSUE THREAD

**Issue by inder2** [microsoft/Microsoft-OpenXR-Unreal#32]
![image](https://user-images.githubusercontent.com/11742609/140543301-322aaa0f-2376-4ac5-898a-10de36d3d69c.png)

**Comment by fieldsJacksonG** [comment]
It looks like you are missing the MicrosoftOpenXR plugin - you should have a "Plugins" directory adjacent to the uproject with a "MicrosoftOpenXR" subdirectory containing the uplugin with those modules.

Try pulling the source again if you accidentally deleted these directories.

Here's a fresh clone of main opened in a binary build of 4.26.2 from the Epic Launcher:
![image](https://user-images.githubusercontent.com/5299526/140583944-67cc497c-1c17-44b5-845a-d5e7584836ea.png)

**Comment by FabianEP11** [comment]
Hi, 
I am trying to open the project but I have the same error (even that I have all of the folders) 
![OpenXR](https://user-images.githubusercontent.com/93542941/163887783-4728b349-28f8-46f9-85e0-94af646bda34.png)

Additionally, if I try to run it from Visual Studio, I'm getting this error:
_Microsoft-OpenXR-Unreal\MsftOpenXRGame\Plugins\MicrosoftOpenXR\Source\MicrosoftOpenXR\Private\HolographicRemotingPlugin.h(13): fatal error C1083: Cannot open include file: 'openxr_msft_holographic_remoting.h ': No such file or directory_
I tried to fix it by adding the openxr.headers / openxr.loader / remoting.openxr in the Manage NuGet Pakages but it didnt work. 

Am I missing something?
Thank you

**Comment by fieldsJacksonG** [comment]
@FabianEP11 Did you follow these instructions to change the plugin to the engine version you are using:
https://github.com/microsoft/Microsoft-OpenXR-Unreal#use-the-plugin-from-github-source

You may need to right click on the uproject after, and select "Switch Unreal Engine version..." to guarantee the sln is targeting the UE version you are using.

**Comment by FabianEP11** [comment]
Yes, I followed the instructions and for some reason I cannot open the project in my computer. 
I tried in a different PC and I got the same error, but on this one when I pressed "yes" on the pop up window for _would you like to rebuild them now_ I, it worked and I was able to open the project. I'm still having issues with the AR session though, so most of the functions are not working properly. 

What could be the issue on my laptop? I've cloned the package at least 4 times and I've tried using the _main_, _bryceh/update_plugin_version_, and _example_project_instructions_ branches, but none of them work for me.

**Comment by uezcr** [comment]
Hi,
I am trying to open the project but I have the same error,too.Only can open with 4.26.2. Neither 4.27.2 nor 5.0.1 could complete the rebuild. Even if select "Switch Unreal Engine version...".
